### PR TITLE
Add pressure-sensing fingers

### PR DESCRIPTION
**Pull request by Adimote**
Adds fingers that fold down and lift up the front of the robot, thus lifting up a cube. The ends of the fingers are pressure sensors, in Arduino analog pins A6 (notably these don't exist on real arduinos), which then return a number proportional to the weight of the robot plus the object being carried.

See Video Demo:
https://user-images.githubusercontent.com/2551763/216791604-d9f9adf9-2754-49e4-b0d4-ab22b4c83f3e.mp4

### CODE REVIEW

File: controllers/example_controller/keyboard_controller.py [hunk 1]
```diff
@@ -122,20 +134,20 @@ def print_sensors(robot: Robot) -> None:
...
-            left_power -= 0.25
-            right_power += 0.25
+            left_power -= 0.125
+            right_power += 0.125
 
         elif key_ascii == key_right:
-            left_power += 0.25
-            right_power -= 0.25
+            left_power += 0.125
+            right_power -= 0.125
```
Comment (PeterJCLaw) [inline controllers/example_controller/keyboard_controller.py:150]: Please can we break this speed change out into a separate PR.

**Comment by PeterJCLaw** [comment]
Numbers I'm seeing:
- 1.0 ish for just the robot
- 1.3 ish for Bronze
- 1.6 ish for Silver
- 1.9 ish for Gold

Given that we're expecting the Gold tokens to weigh about the same as a Silver, this may not be representative.

**Review by WillB97 — APPROVED** [review]

File: controllers/example_controller/keyboard_controller.py (hunk 1 above)
Comment (Adimote) [reply controllers/example_controller/keyboard_controller.py:150]: removed these changes

**Comment by WillB97** [comment]
I get the error:
> WARNING: DEF ROBOT-0 Robot_2022 (PROTO) > DEF FINGERS Transform > DEF FINGER_HINGE HingeJoint > Solid > TouchSensor: Webots has detected that this solid is light and oblong according to its inertia matrix. This belongs in the physics edge cases, and can imply weird physical results. Increasing the weight of the object or reducing its eccentricity are recommended.

**Comment by Adimote** [comment]
> I get the error:
> 
> > WARNING: DEF ROBOT-0 Robot_2022 (PROTO) > DEF FINGERS Transform > DEF FINGER_HINGE HingeJoint > Solid > TouchSensor: Webots has detected that this solid is light and oblong according to its inertia matrix. This belongs in the physics edge cases, and can imply weird physical results. Increasing the weight of the object or reducing its eccentricity are recommended.

now fixed, just had to add 0.05kg :)

**Comment by Adimote** [comment]
I've now split the fingers into two separate servos/pressure sensors. It looks like they normally measure 0.5* the value when they were one, so students can just add them together to get the total pressure.

Also it means I no longer need to tips of the fingers to be 0.2kg, they're now neglegibly light, so the robot is 0.2kg lighter.

**Review by PeterJCLaw — APPROVED** [review]
